### PR TITLE
Minor fixes on the documentation

### PR DESCRIPTION
**Pull request by erickisos**
Some of the links were not currently active (mainly on github side).
There was a link with a minor error on the underscore position.
Added some extra context for some sentences.

### Status

**READY**

### Todo list
- [x] Documentation

### Background context

Just added some fixes that I encountered when I tried to start contributing.

As a side note, if you see something weird with the text or some syntactic problems, just let me know and we can add it to this PR.

### CODE REVIEW

**Review by arthurpaulino — DISMISSED** [review]
Thanks!!

File: docs/source/getting_started.rst [hunk 1]
```diff
@@ -45,13 +51,15 @@ The **transformed training data** is usually just the prediction function applie
...
+Learner functions are usually partially initialized (curried) before being passed to pipelines or applied to data:
+
+.. code-block:: python
 
     from fklearn.training.regression import linear_regression_learner
     from fklearn.training.transformation import capper, floorer, prediction_ranger
 
     # initialize several learner functions
-    capper_fn = capper(columns_to_cap=["income"], precomputed_caps={"income": 50,000})
+    capper_fn = capper(columns_to_cap=["income"], precomputed_caps={"income": 50_000})
```
Comment (gcbeltramini) [inline docs/source/getting_started.rst:62]: Shouldn't this be?

```suggestion
    capper_fn = capper(columns_to_cap=["income"], precomputed_caps={"income": 50000})
```

Comment (erickisos) [reply docs/source/getting_started.rst:62]: afaik [both are valid options](https://peps.python.org/pep-0515/), it is better with your suggestion, but I didn't feel confident enough to remove the separator.
If you think it is readable without the separator, it's ok for me

Comment (gcbeltramini) [reply docs/source/getting_started.rst:62]: I prefer without `_`, because that syntax is not usual.

Comment (erickisos) [reply docs/source/getting_started.rst]: That's a good point, I will add your suggestion then @gcbeltramini

**Review by gcbeltramini — APPROVED** [review]

**Pull request merged**